### ISSUE TITLE
Make error on Alpine for MTConnect cppagent

### ISSUE THREAD

**Issue by skibum1869** [skibum1869/Alpine_MTConnect_Agent#1]
Dependencies

```bash
#11 3.668 (22/47) Installing gcc (9.3.0-r2)
#11 14.07 (23/47) Installing musl-dev (1.1.24-r10)
#11 14.76 (24/47) Installing libc-dev (0.7.2-r3)
#11 14.84 (25/47) Installing g++ (9.3.0-r2)
#11 19.76 (26/47) Installing make (4.3-r0)
#11 23.73 (42/47) Installing cmake (3.17.2-r0)
```

Error output

``` bash
#11 129.6 [100%] Linking CXX static library libdlib.a
#11 129.6 [100%] Built target dlib
#11 129.6 Install the project...
#11 129.6 -- Install configuration: "Release"
#11 129.6 -- Installing: /app_build/install/dlib/lib64/libdlib.a
#11 129.6 -- Installing: /app_build/install/dlib/lib64/cmake/dlib/dlib.cmake
#11 129.6 -- Installing: /app_build/install/dlib/lib64/cmake/dlib/dlib-release.cmake
#11 129.6 -- Installing: /app_build/install/dlib/lib64/cmake/dlib/dlibConfig.cmake
#11 129.6 -- Installing: /app_build/install/dlib/lib64/cmake/dlib/dlibConfigVersion.cmake
#11 129.6 -- Installing: /app_build/install/dlib/lib64/pkgconfig/dlib-1.pc

#11 307.9 make[2]: *** No rule to make target 'install/dlib/lib/libdlib.a', needed by 'agent/agent'.  Stop.
#11 307.9 make[1]: *** [CMakeFiles/Makefile2:316: agent/CMakeFiles/agent.dir/all] Error 2
#11 307.9 make: *** [Makefile:183: all] Error 2
```

This issue is a duplicate of Compilation on x86_64 Linux #156 on the release agent.

Using tukusejssirs recommendation I modified the following code in the cmake/DLib.cmake file from include/dlib/lib to include/dlib/lib64. With this change it compiles fine. So it appears as if the dlib.cmake is pointing to a 32 bit location. 

```cmake
set_target_properties(dlib::dlib PROPERTIES
    IMPORTED_LOCATION "${CMAKE_BINARY_DIR}/install/dlib/lib64/${CMAKE_STATIC_LIBRARY_PREFIX}dlib${CMAKE_STATIC_LIBRARY_SUFFIX}"
    INTERFACE_INCLUDE_DIRECTORIES "${CMAKE_BINARY_DIR}/install/dlib/include"
    )

  if(MSVC)
    set_target_properties(dlib::dlib PROPERTIES
      IMPORTED_LOCATION_DEBUG "${CMAKE_BINARY_DIR}/install/dlib/lib64/${CMAKE_STATIC_LIBRARY_PREFIX}dlibd${CMAKE_STATIC_LIBRARY_SUFFIX}"
      )
  endif()
```

It now compiles but won't run without linking the musl library to run it. See the last line of the command. 

``` bash
/MTC_Agent/agent # .\agent
/bin/sh: .agent: not found

/MTC_Agent/agent # ldd agent
        /lib/ld-musl-x86_64.so.1 (0x7fb0d5349000)
        libstdc++.so.6 => /usr/lib/libstdc++.so.6 (0x7fb0d4e1d000)
        libgcc_s.so.1 => /usr/lib/libgcc_s.so.1 (0x7fb0d4e09000)
        libc.musl-x86_64.so.1 => /lib/ld-musl-x86_64.so.1 (0x7fb0d5349000)

/MTC_Agent/agent # ls /lib
apk                    ld-musl-x86_64.so.1    libattr.so.1           libcrypto.so.1.1       libz.so.1              modules-load.d
firmware               libacl.so.1            libattr.so.1.1.2448    libssl.so.1.1          libz.so.1.2.11         sysctl.d
ld-linux-x86-64.so.2   libacl.so.1.1.2253     libc.musl-x86_64.so.1  libz.so                mdev

/MTC_Agent/agent # file agent
agent: ELF 64-bit LSB shared object, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib/ld-musl-x86_64.so.1, with debug_info, not stripped

/MTC_Agent/agent # /lib/ld-musl-x86_64.so.1 --library-path lib /MTC_Agent/agent agent.cfg
```

**Comment by skibum1869** [comment]
On hold until MTConnect Institute fixes the CMake make issue.

**Comment by skibum1869** [comment]
This will be fixed on the next release version of the MTConnect Adapter when it is fully approved by the SC.

**Comment by skibum1869** [comment]
Closed as this is fixed on 1.7.0.1. This is still broken on 1.7.0.2
